students/luftsmeerflier/lesson03/mailroom.py

```diff
@@ -0,0 +1,182 @@
+#/usr/bin/env python3
+import datetime
+
+donors = [
+	#The actual donations themselves are a list of objects: Donations.date and Donations.amount
+	{
+		"first" : "John",
```

Comment:
in the first version of the mailroom you're expected to use data structures you have learned so far: list and tuples

the recommendation here is to use list of tuples.